Lesson1/T_Ilya/CPP_ADV_Lesson1/functions.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <list>
+#include <forward_list>
+#include <iostream>
+#include <vector>
+
+namespace MFun{
```

Comment:
Namespace is not removed

Reply:
I'm using namespaces to help me understand function local or not, otherwise it's not compiling....